simu_run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Relies on another repository maybe put this script in tat? OR use a git submodule